app/assets/stylesheets/videos.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Videos controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
These extra scss files screw things up. Please delete the new ones. We will be only using application.scss